aviary/subsystems/geometry/test/test_flops_geom_builder.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+import numpy as np
+import openmdao.api as om
+
+from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
+
+from aviary.subsystems.geometry.geometry_builder import CoreGeometryBuilder
+from aviary.variable_info.enums import LegacyCode
+from aviary.variable_info.variables import Aircraft
+from aviary.variable_info.variable_meta_data import _MetaData as BaseMetaData
+import aviary.api as av
+
+FLOPS = LegacyCode.FLOPS
+
+
+class TestFLOPSGeomBuilder(av.TestSubsystemBuilderBase):
+    """
+    That class inherits from TestSubsystemBuilder. So all the test functions are
+    within that inherited class. The setUp() method prepares the class and is run
+    before the test methods; then the test methods are run.
+    """
+
+    def setUp(self):
+        self.subsystem_builder = CoreGeometryBuilder(
+            'core_geometry',
+            BaseMetaData,
+            use_both_geometries=True,
```

Comment:
should this be False for the non hybrid one?

Reply:
Done. Theoretically, we should test all the scenarios of all the booleans on and off. But each case add twenty more tests. Should we do that?

Reply:
Maybe I'm misunderstanding what these tests are for, but I thought TestFLOPSGeomBuilder would be testing the pure FLOPS based geometry, TestGASPGeomBuilder would test the pure GASP based geometry, and then TestFLOPSGeomBuilderHybrid and TestGASPGeomBuilderHybrid would be testing when both are used.

Reply:
I tried it but all unit test fail if `use_both_geometries=False`. I rolled it back.

Reply:
when use_both_geometries is false, `code_origin` needs to be set to either LegacyCode.GASP or LegacyCode.FLOPS